firedrake/adjoint/ensemble_reduced_functional.py

```diff
@@ -41,6 +43,9 @@ class EnsembleReducedFunctional(ReducedFunctional):
     scatter_control : bool
         Whether scattering a control (or a list of controls) over the ensemble communicator
         ``Ensemble.ensemble comm``.
+    gather_functional : ReducedFunctional
```

Comment:
```suggestion
    gather_functional : pyadjoint.ReducedFunctional
```
I think this fixes your doc building fails.